pipelines/concourse.yml

```diff
@@ -1071,131 +1070,6 @@ jobs:
   on_success: *fixed-concourse
   on_failure: *broke-concourse
 
-- name: bosh-prod-deploy
```

Comment:
If you rebase on master now, you'll see that removing this would also mean removing the new `put` step that re-deploys the topgun-worker we have on BOSH. 

https://github.com/concourse/ci/blob/43d09072d69eeecafd547596a0e608bc9941e184/pipelines/concourse.yml#L1107-L1127

Could we rename this job to `prod-deploy` and have it be the job where we re-deploy prod and all it's external workers? I'm not saying do that work in this PR, but setup the home for these future tasks that we know we need to do somewhere.